cpp/include/cudf/functions.h

```diff
@@ -596,6 +596,69 @@ gdf_error gdf_extract_datetime_second(gdf_column *input, gdf_column *output);
 
 /* binary operators */
 
+/**
+ * @brief Binary operation function between gdf_scalar and gdf_column structs.
+ *
+ * The function performs the binary operation of a gdf_scalar operand and a
+ * gdf_column operand.
+ *
+ * The valid field in the gdf_column output will be 1 (by bit) when the two
+ * operands (vax and vay) are not null. Otherwise, it will be 0 (by bit).
+ *
+ * It is required to set in an appropriate manner the fields in the gdf_scalar and
+ * gdf_column structs due to that the binary operation will not be performed.
+ *
+ * @param out (gdf_column) Output of the operation.
+ * @param vax (gdf_scalar) First operand of the operation.
+ * @param vay (gdf_column) Second operand of the operation.
+ * @param ope (enum) The binary operator that is going to be used in the operation.
+ * @return    GDF_SUCCESS if the operation was successful, otherwise an appropriate
+ *            error code
+ */
+gdf_error gdf_binary_operation_s_v(gdf_column* out, gdf_scalar* vax, gdf_column* vay, gdf_binary_operator ope);
```

Comment:
Still feels wrong to call it `lhs` and `rhs` considering that term is used for sides of an equation rather than for operands. But it matches with the terminology used in cuDF.

Reply:
Well, we've got a binary operation where the order matters:

`a op b`

`a` is what's on the "left" of the operation, `'b' is what's on the "right" of the operation. Why would "lhs" and "rhs" be wrong? 

This is exactly the naming scheme used by the C++ standard for things like `operator+` https://en.cppreference.com/w/cpp/numeric/complex/operator_arith3

Reply:
fair point. I was going off of [this SO thread](https://softwareengineering.stackexchange.com/questions/209922/formal-name-for-left-right-hand-side-of-an-expression). I've changed it to `lhs` and `rhs`